src/components/network/use-equipments.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { loadEquipments } from 'redux/actions';
+
+export const useEquipments = (equipmentType) => {
+    const dispatch = useDispatch();
+    const equipments = useSelector(
+        (state) => state.spreadsheetNetwork[equipmentType.resource]
+    );
+    const studyUuid = useSelector((state) => state.studyUuid);
+    const currentNode = useSelector((state) => state.currentTreeNode);
+
+    const isFetching = equipments == null;
```

Comment:
```suggestion
    const isFetching = !equipments;
```

Reply:
Should be the same but for me easier to read

Reply:
fixed